rest/route/host_spawn_test.go

```diff
@@ -364,33 +367,42 @@ func TestModifyVolumeHandler(t *testing.T) {
 		},
 	}
 
-	opts := &model.VolumeModifyOptions{Size: 20, NewName: "my-favorite-volume"}
-	jsonBody, err := json.Marshal(opts)
-	assert.NoError(t, err)
-	buffer := bytes.NewBuffer(jsonBody)
-
-	r, err := http.NewRequest("PATCH", "/volume/volume1/modify", buffer)
-	assert.NoError(t, err)
-	r = gimlet.SetURLVars(r, map[string]string{"volume_id": "volume1"})
```

Comment:
I think it's a good idea to test Parse still

Reply:
restored